tests/docker/Dockerfile

```diff
@@ -135,6 +135,12 @@ COPY --chown=0:0 tests/setup.py /root/tests/
 RUN python3 -m pip install --upgrade --force pip && \
     python3 -m pip install --force --no-cache-dir -e /root/tests/
 
+# Install the OMB tool
+# ToDo: remove checkouting branch when it will be merged to main
+RUN git -C /opt clone https://github.com/redpanda-data/openmessaging-benchmark.git && \
+    cd /opt/openmessaging-benchmark && git checkout ducktape_testing && mvn install -Dlicense.skip=true -q
```

Comment:
it is enough to call `mvn package` instead of `install` we do not need to install omb in local maven repo